dask/dataframe/utils.py

```diff
@@ -445,15 +442,33 @@ def _scalar_from_dtype(dtype):
         raise TypeError("Can't handle dtype: {0}".format(dtype))
 
 
+@make_scalar.register(np.dtype)
+def _(dtype):
+    return _scalar_from_dtype(dtype)
+
+
+@make_scalar.register(pd.Timestamp)
+@make_scalar.register(pd.Timedelta)
+@make_scalar.register(pd.Period)
```

Comment:
Does it make sense for `make_scalar` to handle `pd.Categorical` types? Just thinking about the extensions built into pandas

Reply:
Categorical doesn't have a single scalar type like Timestamp or Period. It's a container for any scalar type, so there's no single type to register.

Reply:
Okay, thanks for clarifying